rust/stackablectl/CHANGELOG.md

```diff
@@ -4,11 +4,16 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+### Changed
+
+- Operators are now installed in parallel when installing a release.
```

Comment:
note: It seems like the reference to the PR is missing here.

Reply:
Sigh.. not sure how that slipped in, cbf36aee41d861cff9b6cb07b2cefafe3f8313f7